docs/ImportONNXDefs.md

```diff
@@ -1,50 +1,116 @@
 <!--- SPDX-License-Identifier: Apache-2.0 -->
+# Table of Contents
```

Comment:
I would add the title of the page at the top: "How to Add an Op"...

Reply:
I will merge first. Fix this small issue later. Do not want to miss the merging opportunity.